python/ray/data/_internal/stats.py

```diff
@@ -558,7 +619,8 @@ def update_execution_metrics(
         force_update: bool = False,
     ):
         op_metrics_dicts = [metric.as_dict() for metric in op_metrics]
-        args = (dataset_tag, op_metrics_dicts, operator_tags, state)
+        per_node_metrics = self._generate_per_node_metrics(op_metrics)
```

Comment:
Let's put these behind a feature-flag that 

 - We keep off by default (for now)
 - We enable at Anyscale to test/verify
 - (If pass previous steps) Enable globally in the next release